mmpose/core/evaluation/top_down_eval.py

```diff
@@ -28,6 +28,8 @@ def _calc_distances(preds, targets, mask, normalize):
     """
     N, K, _ = preds.shape
     distances = np.full((N, K), -1, dtype=np.float32)
+    # handle invalid values
+    normalize[np.where(normalize <= 0)] = 1e6
```

Comment:
use clamp?

Reply:
Note that, I use 1e6 instead of 1e-6 here. 
If `normalize` has invalid values, the normalized distance will be near zero (Any prediction can be regarded as correct.).

Reply:
I see